Project-1/JCheung-P1-README.md

```diff
@@ -0,0 +1,474 @@
+
+6. Explain the difference - using natural language - between the first-order prefixes:
+
+(a) ∃x∀y and ∀x∃y
+
+∃x∀y = "There exists an x such that for every y" or possibly "Something is identical with everything." This can be interpreted as a statement saying that there is at least one value of x for which the statement "for every y" is true. In other words, it asserts the existence of x in relation to all possible values of y.
+
+∀x∃y = "For all x, there exists a y" or possible "Everything is identical with something" This can be interpreted as a statement saying that for every possible value of x, there is at least one corresponding value of y. In other words, it asserts the existence of y in relation to all possible values of x.
+
+(b) ∃x∀y∃z and ∀x∃y∀z 
+
+∃x∀y∃z = "There exists an x such that for every y, there exists a z." This can be interpreted as a statement saying that for some value of x, we can find a corresponding value of z for every possible value of y. It states the existence of x and z in relation to all possible values of y.
+
+∀x∃y∀z = "For all x, there exists a y such that for all z." This can be interpreted as a statement saying that for all possible values of x, we can find a corresponding value of y that holds true for all values of z. In other words, it asserts the existence of y in relation to all possible values of x and z.
+
+(c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+
+∀x∃y∀z∃w = "For all x, there exists a y such that for all z, there exists a w." This can be interpreted as a statement saying that for all possible values of x, we can find a corresponding value of y, and for all possible values of z, we can find a corresponding value of w. In other words, it asserts the existence of y and w in relation to all possible values of x and z.
+
+∃x∀y∃z∀w = "There exists an x such that for all y, there exists a z for all w." This can be interpeted as a statement saying that for some value x, we can find a corresponding value y, and for every y, there is some value z that holds for all of w. 
+	
+7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
+```
+∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+```
+
+```mermaid 
+graph LR
+0(0)
+1(1)
+0 ---0
+0 ---1
+1 ---1
+```
+
+U: {0, 1}
+
+R: {(0. 0), (0, 1), {1, 1)}
+
+Interpretation:
+
+On this interpretation, ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z)) is represented, but ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z)) is not.
+
+On ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z)), everything should R everything and it is not the case in this graph.
+
+
+	
+8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
+
+(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+
+![proof](https://user-images.githubusercontent.com/123851163/216893568-701f0b47-bd92-43fe-9fe7-e6a26d95985a.png)
```

Comment:
What justifies moving from a negated universal in the first line to a negated universal in the second?